.github/workflows/main.yml

```diff
@@ -34,15 +34,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        dotnet: ['net6.0', 'net472', 'net471']
+        dotnet: ['netcoreapp3.1', 'net5.0', 'net6.0', 'net7.0', 'net8.0', 'net462', 'net471', 'net472', 'net48', 'net481']
```

Comment:
.net core 3.1, .net 5 and .net 7 are no longer supported by Microsoft so I would personally drop support for them yourself to reduce maintenance requirements. .net 6 goes out of support in November

Reply:
Thanks for the info, I'll let the team know about this and add it to consideration

Reply:
Both `netcoreapp3.1` and `net5.0` are now removed